Mathlib/Tactic/Ring/RingNF.lean

```diff
@@ -259,9 +259,17 @@ example (x : ℕ) (h : x * 2 > 5): x + x > 5 := by ring; assumption -- suggests
 ```
 -/
 macro (name := ring) "ring" : tactic =>
-  `(tactic| first | ring1 | try_this ring_nf)
+  `(tactic| first | ring1 | try_this ring_nf
+  "\n\nThe `ring` tactic failed to close the goal. Use `ring_nf` to obtain a normal form.
```

Comment:
Depending on the target audience, "obtain a normal form" could still be opaque. (What is a normal form? Ok, I know what a normal form is in general, what's one here? etc.) Is there some other phrasing we could use?

Reply:
Meh, the docstring for `ring_nf` isn't any more informative in this regard. I think it is possible to choose different normal forms though, so specifying which one is used by default is maybe silly. The main reason I added this was just so that an uninformed user would be able to piece together what `nf` means, and why `ring` is suggesting to use that with a `Try this:`.

Reply:
That's a fair point.